src/actions/query.js

```diff
@@ -276,19 +280,18 @@ export function executeQuery(
 						? {
 							from: metaOptions.from,
 							...(value && metaOptions.enableAI === true
-								? { enableAI: true, type: 'search' }
+								? { ...metaOptions, enableAI: true, type: 'search' }
```

Comment:
@SavvyShah Please don't spread the `metaOptions` the query might not accept all the properties present within the metaOptions object.

Reply:
Refer to the previous comment.